src/index.js

```diff
@@ -0,0 +1,140 @@
+          params: { lat: lat, lon: lon }
+        });
+
+        if (weatherResponse.data && weatherResponse.data.main) {
+       
+          const kelvinTemp = weatherResponse.data.main.temp;
+          temperature = Math.round((kelvinTemp - 273.15) * 9/5 + 32);
+          updateDisplay();
+        } else {
+          alert('Unable to fetch weather data.');
+        }
+      } catch (error) {
+        console.error('Error fetching data:', error.message);
+        alert('Unable to fetch temperature. Please try again.');
+      }
+    };
+
+    function updateDisplay() {
+      const tempDisplay = document.getElementById('tempValue');
+      tempDisplay.textContent = `${temperature}°F`;
+      if (temperature >= 80) {
+        tempDisplay.style.backgroundColor = "red";
+        landscape.textContent = "🌡️ BURNING HOT 🌡️\n" +
+          "  ╭──────╮   \n" +
+          " ░│ ☀️ ☀️ │░  \n" +
+          "░░╰──────╯░░\n";
+      } else if (temperature >= 70 && temperature < 80) {
+        tempDisplay.style.backgroundColor = "orange";
+        landscape.textContent = "⛵ SAILING DAY ⛵\n" +
+          "   ⛵ ⛵      \n" +
+          " ∿∿∿∿∿∿     \n" +
+          "∿∿∿∿∿∿∿∿∿\n";
+      } else if (temperature >= 60 && temperature < 70) {
+        tempDisplay.style.backgroundColor = "yellow";
+        landscape.textContent = "⛅ IDEAL DAY ⛅\n" +
+          "   ╭∩∩∩╮    \n" +
+          " ╭∩∩∩∩∩╮   \n" +
+          "∽∽∽∽∽∽∽∽∽\n";
+      } else if (temperature >= 50 && temperature < 60) {
+        tempDisplay.style.backgroundColor = "green";
+        landscape.textContent = "🍂 CRISP DAY 🍁\n" +
+          "  ≈╮__╭≈    \n" +
+          "  ⋯╭━━╮⋯   \n" +
+          " ⋯⋯○⋯⋯⋯   \n" +
+          "⋯⋯⋯⋯⋯⋯⋯\n";
+      } else if (temperature <= 49) {
+        tempDisplay.style.backgroundColor = "teal";
+        landscape.textContent = "🌨️ SNOW DAY ⛄\n" +
+          "  ╭❅╮╭❅╮   \n" +
+          " ╭❅╮╭❅╮╭❅╮  \n" +
+          "▓▓▓▓▓▓▓▓▓\n";
+      }
+    };
+
+    function updateSky() {
+      sky.textContent = "";
+      const skyValue = skySelect.value;
+
+      if (skyValue === "sunny") {
+        sky.textContent = "☁️ ☁️ ☁️ ☀️ ☁️ ☁️" 
+      } 
+      else if (skyValue === "cloudy") {
+        sky.textContent = "☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️"
+      } 
+      else if (skyValue === "rainy") {
+        sky.textContent = "🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧" 
+      } 
+      else if (skyValue === "snowy") {
+        sky.textContent = "🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨"
+      } 
+    };
+
+    skySelect.addEventListener('change', updateSky);
+
+    resetButton.addEventListener('click', () => {
+      cityDisplay.textContent = 'Miami';
+      cityInput.value = 'Miami';
+      updateDisplay();
+      updateSky();
+      getRealTimeTemperature();
+    });
+    
+    increaseTempButton.addEventListener('click', () => {
+      temperature++;
+      updateDisplay();
+    });
+
+    decreaseTempButton.addEventListener('click', () => {
+      temperature--;
+      updateDisplay();
+    });
```

Comment:
Since `increaseTemp` and `decreaseTemp` are very similar, another approach we could take is to combine them into 1 helper function:
```javascript
const setTemp = (changeInTemp) => {
      temperature += changeInTemp;
      updateDisplay();
};
```
Then where we register the handlers, we could call our new function with a parameter by doing so inside an anonymous function:
```javascript
      increaseTempButton.addEventListener('click', () => { setTemp(1) });
      decreaseTempButton.addEventListener('click', () => { setTemp(-1) });
```